nab/detectors/knncad/knncad_detector.py

```diff
@@ -0,0 +1,75 @@
+#################################################
```

Comment:
FYI when this is merged I will add [the Numenta license comments](https://github.com/numenta/NAB/blob/master/nab/detectors/numenta/numenta_detector.py#L1-L19) above.